cirq/contrib/qcircuit/optimizers.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
These aren't really "optimizers", they're just utility methods for putting the circuit into a special form.